common/forms.py

```diff
@@ -89,33 +87,36 @@ class UserForm(BootstrapErrorFormMixin, UserCreationForm):
     first_name = forms.CharField(
         min_length=1,
         max_length=30,
-        label=False,
-        widget=forms.TextInput(attrs={"class": "form-control", "placeholder": "First name*"}),
+        label="First name",
```

Comment:
I think that stars could be useful to highlight fields that are mandatory for every scenario

Reply:
Good point! Now done.